src/pages/studyView/charts/barChart/BarChart.tsx

```diff
@@ -118,6 +119,21 @@ export default class BarChart extends React.Component<IBarChartProps, {}> implem
         ];
     }
 
+    @computed get tilted() {
+        return this.tickValues.length > STUDY_VIEW_CONFIG.thresholds.escapeTick;
+    }
+
+    @computed
+    get bottomPadding(): number {
+        return this.tilted ? adjustedLongestLabelLength(this.tickFormat.map((tick: string | string[]) => {
+            if (_.isArray(tick)) {
+                return tick.join('^');
```

Comment:
Maybe join with an empty string? We never actually show the character `^` in the label. For example, 10^-2 is displayed as 10<sup>-2</sup>. So, I think `10-2` is a better approximation than `10^-2` when calculating the label length.

Reply:
Oh ok, let me do that